README.md

```diff
@@ -87,7 +87,7 @@ server.start();
 or extend the provided HTTP server and override any methods you need:
 ```js
 // my-custom-express-server.js
-const ExpressHTTPServer = require('fastboot-app-server/lib/express-http-server');
+const ExpressHTTPServer = require('fastboot-app-server/src/express-http-server');
```

Comment:
Honestly I'm not super comfortable with people reaching deep into the package to subclass `ExpressHTTPServer`. If we want to do this I think I'd prefer to extract it out into a separate package. Is this a super important use case? I think I'd rather just remove this portion of the README until we have thought more about what a public API would look like.